RELEASES.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 0.6 (1-21-2014)
+- New /validate REST endpoint to probe heapster.
+- Heapster uses InfluxDB service DNS name while running in kube mode.
```

Comment:
I will be making a separate release via github. This is just updating the relevant files.